src/tqec/templates/base.py

```diff
@@ -124,10 +137,18 @@ def shape_instance(self) -> BaseShape:
         """
         return self._shape_instance
 
+    def get_increments(self) -> Displacement:
+        """Get the default increments of the template.
+
+        :returns: a displacement of the default increments in the x and y directions.
+        """
+        return Displacement(self._default_x_increment, self._default_y_increment)
+
 
 @dataclass
 class TemplateWithIndices:
-    """A wrapper around a Template instance and the indices representing the plaquettes it should be instanciated with."""
+    """A wrapper around a Template instance and the indices
+    representing the plaquettes it should be instanciated with."""
```

Comment:
The first line of this docstring was longer than expected from the default Python formatting, but as far as I am aware the documentation tools expect the **first line** to be a quick and short explanation of the class/function/method/... 
Separating that on 2 lines might break the documentation when setup.

Reply:
I though that and just checked. Seems like I was wrong: https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html
I'm fine keeping it with a newline then, this comment is based on incorrect assumptions.